docs/contents.md

```diff
@@ -8,90 +8,84 @@ The **Outbreak Response Framework** solution pack contains the following resourc
 
 ## Picklist
 
-|Name|
-| :- |
-| AlertType |
-| Outbreak Alert Severity |
-| Rule Type |
-| Threat Hunt Tools |
-| Ticketing Tools |
-| Record Status |
+| Name                    | Description |
+|:------------------------|-------------|
+| AlertType               |             |
```

Comment:
Sujit needs to add desc for this artefacts.

Reply:
yes. Awaiting inputs